packages/RatPack.package/RPHandlerMiddleware.class/instance/preHook..st

```diff
@@ -1,7 +1,9 @@
 handler calling
-handle: anEnv next: aNext
+preHook: anEnv
 
 	| response |
 
 	response := (self application perform: anEnv handler withArguments: anEnv parsedRequestArguments).
-	self setBodyIfAppropiate: response in: anEnv.
\ No newline at end of file
+	self setBodyIfAppropiate: response in: anEnv.
+	
+	^ false.
```

Comment:
@He3lixxx you mentioned the idea to put the `#setBodyIfAppropiate` into the `#postHook` (but discarded it because you saw some problems ?!)
If we would write the response directly into the environment at a free place (not `anEnv response`), then  `#setBodyIfAppropiate` could compare those.

I don't know if this is better to separate the code into 2 methods ? I think it is fine like that. 

Now I know what I don't like with the usage of pre and postHook for the HandlerMiddleware.
The preHooks are all methods which are executed before the handler call and the post hook analog after. So isn't that somehow a break ? I think we should implement this in `#handle:next:`

What do you (@all ) think about ?

Reply:
@Nina510 @jakobmache @skydivin4ng3l 

I wrote it this way because I like it this way. We defined a middleware interface and we made a convenience-class to make using that interface easier. It's only consistent to always use our own convenience class in our code, imo (since this really _is_ a middleware).